charts/cromiam/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-cm
```

Comment:
I noticed Greg was using a  `{{ .Values.name }}` for this purpose and I think that is a good pattern. It makes it much easier if we need to make tweaks to application names or override them for some reason down the road.

Reply:
Is the idea that anywhere .Chart.Name would be used, `.Values.name` is substituted? I.e., I shouldn't have any uses of `.Chart.Name` in the chart at all?

Reply:
I think so @gmalkov any thoughts on this?

Reply:
Also, didn't mean for these to be blockers to you merging the pr. I intended them more as general best practice questions 😄